.github/matrix-commitly.yml

```diff
@@ -4,21 +4,35 @@ build-packages:
   os: ubuntu-22.04
   package: deb
   check-manifest-suite: ubuntu-22.04-amd64
+- label: rhel-9
```

Comment:
let's revert the changes in commitly.yml before merge

Reply:
Reverted as the test has passed.